ui/languages/ui_external_languages.scs

```diff
@@ -1,3 +1,9 @@
 ui_external_languages <- sc_node_not_relation;;
 
 ui_external_languages -> scg_code; scs_code;;
+
+nrel_answer
```

Comment:
Why is it here?

Reply:
Merge sources mistake. Removed.
I created separate issue for describing nrel_answer relation: https://github.com/ostis-dev/ims.ostis.kb/issues/94